### PR TITLE
Added antyodaya data +transformer

### PR DESCRIPTION
**Pull request by saurabhlevin**
Added Antyodaya data and shapefiles from bharatmaps. Updated transformer and variables.

### CODE REVIEW

File: Maps/Geojson/od_urban_shapes.geojson
Comment (saurabhlevin) [inline Maps/Geojson/od_urban_shapes.geojson:1]: od_urban_shapes.geojson is required to estimate variables for settlements not present in the Antyodaya dataset. As the source for the original assam_urban_shapes.geojson could not be determined, this shapefile has been approximated by removing the settlements present in the Antyodaya dataset from the Bharatmaps vil2017 layer. This needs to be reviewed.

File: Maps/Geojson/bm_odisha_villages_filtered_reduced.geojson
Comment (saurabhlevin) [inline Maps/Geojson/bm_odisha_villages_filtered_reduced.geojson:1]: Taken from bharat maps vil2017 9k layer. Natural features such as forests have been filtered, and STNAME column have been removed to reduce filesize. The file has been further reduced using geojson-shave

File: Sources/ANTYODAYA/scripts/Transformer.ipynb
Comment (saurabhlevin) [inline Sources/ANTYODAYA/scripts/Transformer.ipynb:1]: As revenue circle maps were not available for Odisha, transformers.py is being run at the subdistrict administrative level.

Splitting the Antyodaya data into "tagged" vs "untagged" dataframes has been commented out as it is not relevant without revenue circle data.

File: Maps/Geojson/bharatmaps_odisha_subdistrict.geojson
Comment (saurabhlevin) [inline Maps/Geojson/bharatmaps_odisha_subdistrict.geojson:1]: As revenue circle shapefiles were not available for Odisha at this stage, bharatmaps_odisha_subdistrict.geojson is being used as a proxy to transform variables at the subdistrict level

**Review by ArcD7 — APPROVED** [review]

**Pull request merged**